src/components/DashboardEditor/DashboardEditorHeader/DashboardEditorHeader.jsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { DocumentImport16, DocumentExport16 } from '@carbon/icons-react';
+
+import { settings } from '../../../constants/Settings';
+import { Button, Breadcrumb, BreadcrumbItem } from '../../../index';
+
+const { iotPrefix, prefix } = settings;
+
+const defaultProps = {
+  breadcrumbs: [],
+  onImport: null,
+  onExport: null,
+};
+
+const propTypes = {
+  /** Dashboard title */
+  title: PropTypes.string.isRequired,
+  /** initial dashboard data to edit */
+  breadcrumbs: PropTypes.arrayOf(PropTypes.node),
+  /** if provided, renders import button linked to this callback */
+  onImport: PropTypes.func,
+  /** if provided, renders export button linked to this callback */
+  onExport: PropTypes.func,
+  /** Callback when cancel button is clicked */
+  onCancel: PropTypes.func.isRequired,
+  /** Callback when submit button is clicked */
+  onSubmit: PropTypes.func.isRequired,
+  /** i18n */
+  i18n: PropTypes.shape({
+    headerImportButton: PropTypes.string,
+    headerExportButton: PropTypes.string,
+    headerCancelButton: PropTypes.string,
+    headerSubmitButton: PropTypes.string,
+  }).isRequired,
+};
+
+const DashboardEditorHeader = ({
```

Comment:
Could you add a custom `extraContent` or `extraItems` prop so that I can add any extras that I need here?